arch/arm/core/cortex_m/Kconfig

```diff
@@ -65,7 +65,7 @@ config ARMV6_M
 	select ATOMIC_OPERATIONS_C
 	select ISA_THUMB2
 	help
-	  This option signifies the use of an ARMv6-M processor implementation.
+	  This option signifies the use of an Armv6-M processor implementation.
```

Comment:
Has the case been changed for v6 and v7? Ithought the correct spelling were:
* ARMv6-M
* ARMv7-M
* Armv8-M

Reply:
Please, check https://developer.arm.com/products/architecture/m-profile. It seems that ARM has moved to the new style/nomenclature, regarding the Architecture namings.

Reply:
While ARM has rebranded to Arm, I don't think it makes sense to make such changes in our code base.  I'm sure we are always going to have a mix, and I'm not keen on trying to match arm's marketing department on how to capitalize the letters in 'ARM'

Reply:
Makes sense. That is why I made the changes only in this specific file, because different spelling styles within the same file might cause some confusion. 

But we could revert these changes if you insist. No strong opinion on that.

Reply:
lets revert such changes.

Reply:
Will do.